integration/test_collection.py

```diff
@@ -1657,11 +1665,14 @@ def test_sort(
             Property(name="name", data_type=DataType.TEXT),
         ],
     )
-    uuids_from = [
-        collection.data.insert(properties={"name": "A", "age": 20}),
-        collection.data.insert(properties={"name": "B", "age": 22}),
-        collection.data.insert(properties={"name": "C", "age": 22}),
-    ]
+
+    uuid1 = collection.data.insert(properties={"name": "A", "age": 20}, uuid=str(UUID1)[:-1] + "1")
+    time.sleep(1)
+    uuid2 = collection.data.insert(properties={"name": "B", "age": 22}, uuid=str(UUID1)[:-1] + "2")
+    time.sleep(1)
+    uuid3 = collection.data.insert(properties={"name": "C", "age": 22}, uuid=str(UUID1)[:-1] + "3")
+
+    uuids_from = [uuid1, uuid2, uuid3]
```

Comment:
1s sleep is a bit long, is that even needed?

Reply:
It's to make sure that the times comparisons don't flake. Is there a better way of doing it, I'll reduce the time it if not